### PR TITLE
Nalin

### PR DESCRIPTION
**Pull request by nalin97**

### CODE REVIEW

File: server/back-end/models/wordmodel.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+const sql = require("./db.js");
+const { response } = require("express");
+
+// Word
+const Word = function (word) {
+    this.word_name = word.name;
+    this.created_at = Date.now()// add new timestamp ;
+    this.updated_at = Date.now()// add new timestamp ;
+    this.use_yn = word.use_yn;
```
Comment (MichelleTanzil) [inline server/back-end/models/wordmodel.js:9]: Can you add a comment as to what use_yn is?

File: server/back-end/models/wordmodel.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    this.word_icon = word.word_icon;
+    this.category_idx = word.category_idx;
```
Comment (MichelleTanzil) [inline server/back-end/models/wordmodel.js:11]: According to this, is there table for categories as well?

File: server/back-end/models/wordmodel.js (hunk 1 above)
Comment (daesung626) [reply server/back-end/models/wordmodel.js:9]: use_yn is to determine if we're using the word or not (yes or no). Think of it as a boolean

File: server/back-end/models/wordmodel.js (hunk 2 above)
Comment (daesung626) [reply server/back-end/models/wordmodel.js:11]: Yes

Comment (MichelleTanzil) [reply server/back-end/models/wordmodel.js:11]: Got it, can you/nalin complete that by the end of the week or is more time needed?

File: server/back-end/models/wordmodel.js (hunk 1 above)
Comment (MichelleTanzil) [reply server/back-end/models/wordmodel.js:9]: Can you give me more context on this use_yn? Why and how is it being used?

**Comment by MichelleTanzil** [comment]
@nalin97 Can you work on this branch to resolve and merge it?

**Comment by MichelleTanzil** [comment]
This merge request is approved.

**Pull request merged**